pallets/base-nft/src/lib.rs

```diff
@@ -279,22 +286,33 @@ impl<T: Config> Pallet<T> {
 
 			let token_info = TokenInfo {
 				metadata,
-				owner: owner.clone(),
+				owners: owners.into_iter().cloned().collect(),
 				data,
 			};
+
+			let owners_token = token_info.owners.clone();
```

Comment:
I'd like to avoid doing this. What I'm doing here is making sure `token_info.owners` is not moved before it's used below.